### DISCUSSION TITLE
Docker command line (monitor / synchronize)

### DISCUSSION THREAD

**Discussion by darrepac** [abraunegg/onedrive#1433] [Q&A] [answered]
Hi

The default docker command line is
 `/usr/local/bin/onedrive --monitor --confdir /onedrive/conf --syncdir /onedrive/data
`
Here is what happened:
- onedrive linux client started to download all my files in my local dir (good)
- in the meantime I did some cleanup in my files from an other PC, and changes are well done by checking onedrive online
- onedrive linux client finished the process of downloading after some hours, without reflecting most of my changes

Now there is regularly in the docker logs:
```
Syncing changes from OneDrive ...
Sync with OneDrive is complete
```

If I delete a file from an other PC, the linux client delete it as well... but all the changes done previously are not reflected.

Is this a normal behaviour?
What did I miss?
The default onedrive linux client should not use the "--synchronize" option?

thanks

**Comment by abraunegg** [comment, marked as answer]
@darrepac 
`--synchronize` is for a single one time sync. `--monitor` is for ongoing sync, checking OneDrive for changes, syncing data every 300 seconds.

> If I delete a file from an other PC, the linux client delete it as well... but all the changes done previously are not reflected

All your changes, as reflected by OneDrive *should* be reflected locally. If however, you are also using other clients (rclone etc) these impact the change sequence number, thus, changes will be missed.

**Comment by darrepac** [comment]
Ok
I don’t have other linux client (I got enough difficulty to make this one working ;) )
May-be it is due to the fact that I made the changes while it was downloading the whole drive (?)

How can I force it to take a deep look and make the changes? Restart the client?

**Reply by abraunegg** [reply]
@darrepac 
Please review the documentation for Docker - you want to initiate a resync: https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md#usage-examples

**Reply by darrepac** [reply]
Well, I am doing a resync but it is uploading files that I deleted from other places previously... let's say the initial state was not correct and let see if it goes well after... (I have an other problem, but will do a new discussion)

**Reply by abraunegg** [reply]
@darrepac 
> Well, I am doing a resync but it is uploading files that I deleted from other places previously... let's say the initial state was not correct and let see if it goes well after... (I have an other problem, but will do a new discussion)

A resync removes the local state, thus, will upload / download all files that it needs to.

Please read the documentation on how to force different modes of operation.
